Gemfile

```diff
@@ -27,9 +27,14 @@ gem "turbolinks", "~> 5"
 gem "webpacker", "~> 4.0"
 
 group :development, :test do
+  gem "database_cleaner"
```

Comment:
Gem này để làm gì

Reply:
Dạ Gem này dùng để xóa sạch toàn bộ cơ sở dữ liệu sau mỗi lần chạy test, sau khi chạy xong một test nó sẽ rollback database lại trạng thái trước khi chạy test.